cores/arduino/WInterrupts.h

```diff
@@ -31,7 +31,7 @@ extern "C" {
 #define FALLING 3
 #define RISING 4
 
-#define DEFAULT 1
+//#define DEFAULT 1
```

Comment:
@ladyada like nRF the Arduino TFL or its deps define DEFAULT symbol which cause the conflict here. I tried but didn't know if this symbol is used anywhere. Please let me know if any of ours lib use this symbol or it can be safely commented out

Reply:
i think its ok to remove this for now, i saw this issue too :( its a bad #define! :)